array.ts

```diff
@@ -345,6 +346,23 @@ export const unzip = <A, B>(
   return [fa, fb];
 };
 
+/**
+ * Returns a new array containing elements of `as` sorted in ascending order
+ * according to the sort order defined by `O`.
+ *
+ * @example
+ * import { ordNumber } from "https://deno.land/x/fun/ord.ts";
+ * import { sort } from 'https://deno.land/x/fun/array.ts'
+ *
+ * sort(ordNumber)([3, 1, 2])
+ * // [1, 2, 3]
+ *
+ * @category combinators
+ */
+export const sort = <B>(O: Ord<B>) =>
+  <A extends B>(as: ReadonlyArray<A>): ReadonlyArray<A> =>
+    as.slice().sort(compare(O));
```

Comment:
This is good but I have two nitpicks:

1. I'd like to move towards exporting full functions instead of arrow functions. I know there are arrow function exports in array.ts already but style-wise those will be migrated to functions for the v2.0 release.
2. For a very very small perf gain I'd like to see the comparison function created before the sort function return.

```ts
export function sort<B>(O: Ord<B>): <A extends B>(as: ReadonlyArray<A>) => ReadonlyArray<B> {
  const _compare = compare(O);
  return as => as.slice().sort(_compare);
}
```